pkg/lib/config/configsource/resources.go

```diff
@@ -101,7 +101,8 @@ func (d AuthgearYAMLDescriptor) ViewResources(resources []resource.ResourceFile,
 			return nil, err
 		}
 
-		appConfig, err := config.Parse(bytes.([]byte))
+		// FIXME: the context should be passed from the caller
```

Comment:
We should update the interface of Descriptor to make all methods take ctx as the first argument.

Reply:
Maybe I will only update `ViewResources` in this pr, because other methods seems not affected and require a lot of changes.